test/integration/globals/app-nav.js

```diff
@@ -177,9 +175,8 @@ context('App Nav', function() {
       .get('.picklist')
       .contains('Sign Out')
       .click()
-      .then(() => {
-        expect(logoutStub).to.have.been.calledOnce;
-      });
+      .url()
+      .should('contain', 'logout');
```

Comment:
The `Sign Out` item was recently changed to `<a href="/logout">` instead of `Radio.request('auth', 'logout');` (https://github.com/RoundingWell/care-ops-frontend/commit/d579669a443924af222d65e63293477346d97462).

So the `logoutStub` we were using in this test didn't work anymore.

Reply:
Okay makes sense, I switched that over to check the `href` value.